src/app/applications/[id]/page.tsx

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import ApplicationDetailsContainer from "@/components/applicationDetailsContainer";
+import Button from "@/components/button";
+import FieldAttachmentContainer from "@/components/fieldAttachmentContainer";
+
+type ApplicationDetailsPageProps = {
+  params: { id: string };
+};
+
+export default function ApplicationDetailsPage({
+  params,
+}: ApplicationDetailsPageProps) {
+  const { id } = params;
+  return (
+    <div className="mt-20 grid grid-cols-12 gap-x-20">
+      <div className="col-span-6 col-start-3">
+        <div className="px-3">
+          <div className="flex justify-between">
+            <h1 className="text-3xl text-primary">Application Title</h1>
+            <div className="flex gap-x-3">
+              <Button type="warning" text="Save changes" className="text-xs" />
```

Comment:
important: I really liked icons on the left side of action buttons - I think all of them have. Can you please add to these two as well?